README_OSE.md

```diff
@@ -128,5 +144,17 @@ the directory as the inventory) use the -i option for ansible-playbook.
 On the master host:
 ```sh
 systemctl restart openshift-sdn-master
-openshift ex router --create=true --credentials=/var/lib/openshift/openshift.local.certificates/openshift-client/.kubeconfig
+openshift ex router --create=true \
+  --credentials=/var/lib/openshift/openshift.local.certificates/openshift-client/.kubeconfig \
+  --images='docker-buildvm-rhose.usersys.redhat.com:5000/openshift3_beta/ose-${component}:${version}'
```

Comment:
Did these not pull correctly?  The /etc/sysconfig/openshift-{master,node} should be using the docker-buildvm host as the default registry (https://github.com/detiber/openshift-ansible/blob/enterprise2/inventory/byo/group_vars/all#L17)